header-validator/src/validate-json.ts

```diff
@@ -296,12 +296,21 @@ const filterData = () =>
       state.error('is prohibited because it is implicitly set')
       return
     }
+    if (filter === '_lookback_window') {
+      state.error('is prohibited because it would be ignored')
```

Comment:
Is there a reason to use a different message for this compared to `source_type`?

Reply:
The idea was that we are not really comparing to a generated value, but we could probably consider it to be implicitly set. I updated.